products/micronaut.md

```diff
@@ -27,7 +34,7 @@ releases:
 -   releaseCycle: "2"
     releaseDate: 2020-06-26
     eol: false
-    support: false
+    support: 2023-03-01
```

Comment:
So, the "Maintenance Support Through" date is actually the end of Maintenance Support interval. Basing this on [the Nov 2022 snapshot](http://web.archive.org/web/20221126191326/https://micronaut.io/support-schedule/) which also shows v2 in "Active Maintenance".

It was marked under maintenance in August 2022, which is the first snapshot on the Web Archive. Looking at [2.5.x](https://github.com/micronaut-projects/micronaut-core/commits/2.5.x) releases, the switch from "features->bug fixes only" happened around August 2021, which is when v3 was released.

So I'm guessing:

support(x) = releaseDate(x+1) #Active Development
eol(x) = As per the website, which gets extended ocassionally

Hence, suggested:
```suggestion
    support: 2021-08-18
```

Reply:
Good finding :+1: